templates/arnold/render-windows-frames/job.template.json

```diff
@@ -51,7 +51,7 @@
                 },
                 "repeatTask": {
                     "displayName": "frame {fileNameWithoutExtension}",
-                    "commandLine": "mkdir images & %ARNOLD_2018_EXEC% -l \"C:\\server\\mayaArnold\\2018\\shaders\" -i \"%AZ_BATCH_TASK_WORKING_DIR%\\assets\\{fileName}\" -dw -v 6 -o \"%AZ_BATCH_TASK_WORKING_DIR%\\images\\{fileNameWithoutExtension}.jpeg\"",
+                    "commandLine": "mkdir images & %ARNOLD_2020_EXEC% -l \"C:\\server\\mayaArnold\\2018\\shaders\" -i \"%AZ_BATCH_TASK_WORKING_DIR%\\assets\\{fileName}\" -dw -v 6 -o \"%AZ_BATCH_TASK_WORKING_DIR%\\images\\{fileNameWithoutExtension}.jpeg\"",
```

Comment:
\"C:\\server\\mayaArnold\\2018\\shaders\ is this directory correct for 2020?

Reply:
can be updated to 2020